src/main/java/com/purbon/kafka/topology/model/impl/ProjectImpl.java

```diff
@@ -42,7 +41,7 @@ public ProjectImpl(String name) {
     this(name, new TopologyBuilderConfig());
   }
 
-  public ProjectImpl(String name, TopologyBuilderConfig config) {
+  public ProjectImpl(String name, TopologyBuilderConfig builderConfig) {
```

Comment:
why the name change?

Reply:
This class is used in many places. There are some other classes, as TopicImpl, that contain a `config` attribute already. For consistency, I suggest to name `TopologyBuilderConfig` variables the same across clients. 
This was helpful when debugging some tests.